src/scripts/addUser.ts

```diff
@@ -0,0 +1,37 @@
+import bcrypt from 'bcryptjs';
+import db from 'src/db';
+
+const addUser = async (
+  email: string,
+  password: string,
+  gestionnaire: string
+) => {
+  const user = await db('users').select().where({ email }).first();
+
+  if (user) {
+    console.error(`email ${email} already exists`);
+    process.exit(1);
+  }
+  const salt = await bcrypt.genSalt(10);
+
+  await db('users').insert({
+    email,
+    password: bcrypt.hashSync(password, salt),
+    gestionnaire,
+  });
+  console.info('User created');
+  process.exit(0);
+};
+
+if (process.argv.length !== 5) {
+  console.info(
+    'Usage: export NODE_PATH=./ && ts-node src/scripts/addUser.ts email password gestionnaire'
```

Comment:
In my case, I've got this error when i run this command : 
`ts-node: not found`

For fixing, I have run: `export NODE_PATH=./ && npx ts-node src/scripts/addUser.ts email password gestionnaire`